cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -124,6 +126,13 @@ object ReporterCommand : CommandWithHelp() {
     )
     private var customLicenseTextsDir: File? = null
 
+    @Parameter(
+        description = "A YAML that contains the license configuration.",
```

Comment:
"A YAML **file**...". Or remove "YAML", because it could also be JSON or XML, and we don't mention the format in other parameters like `--resolutions-file` or `--copyright-garbage-file`.

Reply:
Done.